code/game/gamemodes/changeling/changeling.dm

```diff
@@ -366,11 +366,12 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 
 	var/datum/changelingprofile/prof = new()
 
-	H.dna.real_name = H.real_name //Set this again, just to be sure that it's properly set.
```

Comment:
This seems to be causing an "undefined variable" error which is why you couldn't absorb through hivemind in the first place. I tested absorbing normally and through hivemind and both are working